modin/pandas/series.py

```diff
@@ -484,7 +484,10 @@ def values(self):  # noqa: RT01, D200
         """
         Return Series as ndarray or ndarray-like depending on the dtype.
         """
-        return self.to_numpy()
+        data = self.to_numpy()
+        if isinstance(self.dtype, pandas.CategoricalDtype):
+            data = pandas.Categorical(data, dtype=self.dtype)
```

Comment:
Can we use "our own" categorical class somehow? I know that `modin.pandas.Categorical` is just an alias for the pandas class, but maybe it would change in the future and so then we wouldn't need to rename this all over the project.
```suggestion
        if isinstance(self.dtype, pd.CategoricalDtype):
            data = pd.Categorical(data, dtype=self.dtype)
```